launcher/ui/pages/global/AccountListPage.cpp

```diff
@@ -236,8 +236,8 @@ void AccountListPage::updateButtonStates()
     }
     ui->actionRemove->setEnabled(accountIsReady);
     ui->actionSetDefault->setEnabled(accountIsReady);
-    ui->actionUploadSkin->setEnabled(accountIsReady && accountIsOnline);
-    ui->actionDeleteSkin->setEnabled(accountIsReady && accountIsOnline);
+    //ui->actionUploadSkin->setEnabled(accountIsReady && accountIsOnline);
```

Comment:
This will also disable these buttons for Microsoft accounts.

Reply:
Then should I do it like this?
```
ui->actionUploadSkin->setEnabled(accountIsReady && accountIsOnline && !accountIsElyby);
ui->actionDeleteSkin->setEnabled(accountIsReady && accountIsOnline && !accountIsElyby);
```
and ofc add `accountIsElyby` property.

Reply:
I would modify the `accountIsOnline` variable and add the MSA check there - remember, keep the patch as small as possible for easy updates

Reply:
`accountIsOnline` is a boolean variable. Will it break offline account creation if I change it to false when the user adds an Ely.by account?

Reply:
I didn't understand your point, but changing that variable will break account refresh. So you'll have to add another variable (preferably `accountIsMSA`)